setup.py

```diff
@@ -33,7 +33,9 @@
       author_email='roel.vandenberg@nelen-schuurmans.nl',
       url='http://demo.lizard.net',
       license='GPL',
-      packages=['lizard_connector'],
+      packages=find_packages(exclude=[
```

Comment:
Er is maar één top-level package, namelijk lizard_connector. Dus de oorspronkelijke simpele regel was prima.

Reply:
Je kunt proberen @reinout , maar bij mij werkt het niet in ieder geval.

Reply:
mea culpa @reinout , lijkt toch *wel* te werken zonder find_packages. Had denk ik wat overgebleven bestanden over die niet werden opgeschoond.